src/components/Chart.jsx

```diff
@@ -137,4 +139,15 @@ export default connect(({ chart, drawTools, studies, chartSetting, chartType })
     chartPanelTop: chart.chartPanelTop,
     setting: chartSetting,
     chartContainerHeight: chart.chartContainerHeight,
+     hasOpenMenu: (
```

Comment:
@MahboobehMohammadi its not about this issue, but please check by @brucebinary  whether its good to pass `hasOpenMenu` to `<ChartControls hasOpenMenu={hasOpenMenu} .../>` and use it there. because we do the same in that component (prevent duplicate code)

Reply:
@shayanbinary good point